### PR TITLE
Draft proposal to add Zcash (ZEC) to the Celo Reserve

### PR DESCRIPTION
**Pull request by alchemydc**
I believe that this will promote engagement between the Zcash and Celo communities, which are ideologically very much aligned.  Down the track I hope to encourage development of automated, trustless bridges between the two protocols, which would enable automated, transparent management of the Zcash component of the reserve.  I also believe diversifying the Celo Reserve with ZEC will help to mitigate the extent to which the reserve is presently exposed to Ethereum risk.

### CODE REVIEW

**Comment by alchemydc** [comment]
@martinvol PR opened per your request in Discord.  Keen to get your comments. Thanks!

File: CGPs/0009.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+This proposal advocates that the target allocation of the Celo reserve be modified as follows:
+* 50% CELO
+* 30% BTC
+* 10% ETH
+* 5% DAI
+* 5% ZEC
+
+
+## Verification
+This proposal involves no code changes or verification.
```
Comment (martinvol) [inline CGPs/0009.md:57]: It should change `assetAllocationSymbols` I believe, see https://github.com/celo-org/celo-monorepo/blob/8a8b82252e2da9a4afc743b1e27260d521729f74/packages/protocol/migrationsConfig.js#L153

File: CGPs/0009.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+* 30% BTC
+* 15% ETH
+* 5% DAI
+
+This proposal advocates that the target allocation of the Celo reserve be modified as follows:
+* 50% CELO
+* 30% BTC
+* 10% ETH
+* 5% DAI
+* 5% ZEC
```
Comment (martinvol) [inline CGPs/0009.md:53]: I like the idea of diversifying the assets in the reserve further, specially with assets that have different features and could enable new use-case shall a bridge be created. 

As the primary role for the reserve is to provide collateral for cUSD, I'd think the community would be interested about the economics of Zcash, like what's the expected circulation supply in the near future and what factors are key to increase its value. I personally consider fundamental analysis for every asset wanted to be added to the reserve a must.

Is there some document that you can provide regarding this?

**Comment by martinvol** [comment]
@alchemydc sorry it took me so long to reply!

File: CGPs/0009.md (hunk 2 above)
Comment (alchemydc) [reply CGPs/0009.md:53]: Hi Martin,

Thanks for the feedback.

Zcash follows the same issuance curve as Bitcoin.  The first Zcash halvening is [coming up in November](https://www.nicehash.com/countdown/ZEC-halving-2020-11-17-12:00#!).

One particularly large factor to consider is that the Zcash community [recently decided](https://electriccoin.co/reaching-consensus/) to allocate 20% of issuance for the four years following the halvening to a development treasury.  Notably, the largest slice of this treasury (8% of issuance) is allocated to provide funding for third parties to develop and extend the Zcash platform and ecosystem.

Another factor to consider is that the ECC team continues to [develop and release innovative cryptography](https://www.coindesk.com/zcashs-halo-breakthrough-is-a-big-deal-not-just-for-cryptocurrencies) that solves problems that need to be solved for Zcash specifically, and also blockchains more generally.

Here are some links that provide additional perspective on Zcash for the Celo community:

[Placeholder Zcash Thesis](https://www.placeholder.vc/blog/2020/9/15/zcash-thesis)
[Zcash Analysis by BraveNewCoin](https://bravenewcoin.com/insights/zcash-price-analysis-transactions-per-day-continues-to-rise)
[Messari Profile](https://messari.io/asset/zcash/profile)

DC

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+*.code-workspace
```
Comment (YazzyYaz) [inline .gitignore:1]: Can we remove this part of the .gitignore from this PR?

File: CGPs/0015.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
+# CGP [0015]: Add Zcash (ZEC) to Celo Reserve
+
+- Date: 07 July 2020, updated 30 Oct 2020
+- Author(s): @alchemydc
+- Status: DRAFT
+- Governance Proposal ID #: GCP [0015]
+- Date Executed: TBD
```
Comment (YazzyYaz) [inline CGPs/0015.md:7]: Can you add a Discussions link? something like 
```
- Discussions-To: https://forum.celo.org/path-to-forum-post-on-proposal
```

**Review by YazzyYaz — CHANGES_REQUESTED** [review]

**Review by prestwich — CHANGES_REQUESTED** [review]
To be merged as draft:
1. Open a discussions issue `CIP XX - Short Title - Discussions`
2. Update the discussions-to link in the CIP header

**Comment by YazzyYaz** [comment]
@alchemydc from James feedback, please create a discussions link, either here on Github or on the Forum, then link to it here in the CGP. 

In the preamble, you can just add:
```
Discussions To: <URL TO DISCUSSIONS>
```

**Pull request closed without merging**